src/utils/Parsing.fs

```diff
@@ -399,28 +412,28 @@ module internal Implementation =
                           finished <- true;                                                                             
                           valueStack.Push(ValueInfo(res, lhsPos.[0], lhsPos.[1])) 
                     | RecoverableParseError ->
-#if DEBUG
+#if __DEBUG
                           if Flags.debug then Console.WriteLine("RecoverableParseErrorException...\n");
 #endif
                           popStackUntilErrorShifted(None);
                           // User code raised a Parse_error. Don't report errors again until three tokens have been shifted 
                           errorSuppressionCountDown <- 3
                 elif kind = errorFlag then (
-#if DEBUG
+#if __DEBUG
```

Comment:
it's ok, but with a better name like `SHOW_DEBUG_INFO`, i'll update it
if we use `DEBUG`, we cannot opt in the info on `RELEASE` or disable it in `DEBUG` configuration